pilot/pkg/model/jwks_resolver_test.go

```diff
@@ -292,6 +292,7 @@ func TestJwtPubKeyEvictionForNotRefreshed(t *testing.T) {
 		for {
 			select {
 			case <-done:
+				c.Stop()
```

Comment:
nit: why not defer, so we will never forget `Stop` it

Reply:
You are right, it is more reasonable to use defer here